retail/retail-java-applications/data-engineering-dept/business-logic/src/main/java/com/google/dataflow/sample/retail/businesslogic/core/transforms/clickstream/ClickStreamSessions.java

```diff
@@ -85,7 +86,8 @@ public ClickStreamSessions withSessionWindowGapDuration(Duration sessionWindowGa
      * }</pre>
      */
     return input
-        .apply(Window.into(Sessions.withGapDuration(getSessionWindowGapDuration())))
-        .apply(Group.byFieldNames("client_id"));
+        .apply("SessionWindow",
```

Comment:
As a pipeline may have potentially > 1 SessionWindow , what about ClickStreamSessionWindow or CSSessWin ?

Reply:
Addressed in a new commit.